doc/internal/release-github.md

```diff
@@ -0,0 +1,81 @@
+The libcgroup Release Process
+===============================================================================
+https://github.com/libcgroup/libcgroup
+
+This is the process that should be followed when creating a new libcgroup
+release.
+
+#### 1. Verify that all issues assigned to the release milestone have been resolved
+
+  * https://github.com/libcgroup/libcgroup/milestones
+
+#### 2. Verify that the Github Actions are all passing
+
+#### 3. Verify that the bundled test suite runs without error
+
+	# ./bootstrap.sh
+	# make check
+
+#### 4. Verify that the packaging is correct
+
+	# make distcheck
+
+#### 5. Perform any distribution test builds
+
+  * Oracle Linux
+  * Fedora Rawhide
+  * Red Hat Enterprise Linux
+  * etc.
+
+#### 6. If any problems were found up to this point that resulted in code changes, restart the proces
```

Comment:
@drakenclimber The patch set looks great. A minor typo, `proces` -> `process`.

Reply:
Fixed.  Thanks